homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -32,6 +37,22 @@
     vol.Optional(CONF_CODE): cv.string,
     vol.Optional(CONF_PENDING_TIME, default=DEFAULT_PENDING_TIME):
         vol.All(vol.Coerce(int), vol.Range(min=0)),
+    vol.Optional(STATE_ALARM_ARMED_AWAY): {
+        vol.Optional(CONF_PENDING_TIME):
```

Comment:
Break out and make a separate schema for this part that you can reuse for the four states.

Reply:
will make the change